community-members.md

```diff
@@ -55,6 +55,7 @@ repositories:
 - [Cijo Thomas](https://github.com/cijothomas), Microsoft
 - [Daniel Dyla](https://github.com/dyladan), Dynatrace
 - [David Ashpole](https://github.com/dashpole), Google
+- [Leighton Chen](https://github.com/lzchen), Microsoft
```

Comment:
Perhaps need to remove me from emeritus sponsors as well?

Reply:
+1, and for emeritus, we should probably remove company info as we don't expect folks to maintain it? Not for this PR though.

Reply:
I sent a small PR for this #2381.